matlab_simulation/code v1.0/ME597-7-Trajectories/motion_model.m

```diff
@@ -0,0 +1,13 @@
+function xddot = motion_model(t, T, xd)
```

Comment:
`v` and `w` both should be input variables to the motion model, also consider giving this function a better name other than `motion_model()`, I assume this is for a two wheeled robot?